### ISSUE TITLE
Better Multi-Spin does not appear to do anything in v1.4.2-beta

### ISSUE THREAD

**Issue by QuackQuackBlah** [PoochyEXE/PoochyEXE.github.io#20]
It still seems to subtract the whole 10% spin value even though in the description it states that it refunds the additional spins. This issue seem to occur both with manual spin and with auto-spin

**Comment by PoochyEXE** [comment]
Looks like I broke this in the v1.2.0-beta update. Fixed in v1.4.3-beta. Thanks for the bug report!

**Comment by PoochyEXE** [comment]
Fun behind-the-scenes story: One of the things I did to test the v1.2.0-beta update was to import a save from v1.1.1 with everything maxed out into two tabs, one running v1.1.1 and one running v1.2.0-beta and leave them running. Eventually I noticed that v1.2.0-beta was gaining points slightly about 5%~10% slower, but I couldn't figure out why. I double-checked all the point values from balls, spins earned by balls, point values awarded by the green, cyan, and blue bonus wheel spaces, and still missed this bug. Also asked a couple friends to see if they could spot anything wrong with v1.2.0-beta, and none of them did.

Seriously, nice catch on this one!
